shared/domains/chelonia/localSelectors.js

```diff
@@ -28,57 +28,59 @@ export default (sbp('sbp/selectors/register', {
     reactiveSet: Function,
     reactiveDel: Function
   }) => {
-    sbp('okTurtles.events/on', EVENT_HANDLED, async (contractID, message) => {
-      const { contractState, cheloniaState } = await sbp('chelonia/contract/fullState', contractID)
-      const externalState = sbp(stateSelector)
-      if (cheloniaState) {
-        if (!externalState.contracts) {
-          reactiveSet(externalState, 'contracts', Object.create(null))
+    sbp('okTurtles.events/on', EVENT_HANDLED, (contractID, message) => {
+      sbp('okTurtles.eventQueue/queueEvent', EVENT_HANDLED, async () => {
```

Comment:
So what is going on with this `EVENT_HANDLED`...? It started out as an event, and now it seems to have evolved into a new queue.

Questions:

1. Why is it a queue now?
2. Why is it being used here as a queue immediately after we handle the event?
3. Is there a different, existing queue you could use instead?

If you can't get rid of this `EVENT_HANDLED` queue, and you are *positive* it is needed, then could you please add a comment here explaining all of this?